examples/frontend/pages/share/explore/index.tsx

```diff
@@ -0,0 +1,179 @@
+// @ts-nocheck
```

Comment:
<img width="468" alt="Screenshot 2021-11-18 at 15 14 25" src="https://user-images.githubusercontent.com/5102099/142431793-e415cb6c-1605-4219-b5e3-b062a2e4ade1.png">

The spacing is a bit off on mobile.